itou/templates/approvals/declare_prolongation.html

```diff
@@ -113,4 +125,23 @@ <h5 class="card-title">
     {{ block.super }}
     <!-- Needed to use the Datepicker JS widget. -->
     {{ form.media }}
+
+    {# Clicking on a reason that does not require the opinion of a prescriber hide email field  #}
+    <script type="text/javascript">
+        $(document).ready(() => {
+            var reasons_not_need_prescriber_opinion = ["{{ form.reasons_not_need_prescriber_opinion|join:'", "' }}"];
```

Comment:
Je te suggère d'implémenter ce filtre ça nous sera bien utile je pense :) 
```py
// itou/templatetags/dict_filters.py ou js_filters.py

from django.utils.safestring import mark_safe
from django.template import Library

import json


register = Library()


@register.filter(is_safe=True)
def js(obj):
    return mark_safe(json.dumps(obj))
```

```suggestion
            var reasons_not_need_prescriber_opinion = {{ form.reasons_not_need_prescriber_opinion|js }};
```

Reply:
ah oui, ça peut être pratique ;)